LEGO1/lego/legoomni/src/act1/act1state.cpp

```diff
@@ -1,34 +1,295 @@
 #include "act1state.h"
 
+DECOMP_SIZE_ASSERT(Act1State, 0x26c)
+DECOMP_SIZE_ASSERT(Act1State::NamedPlane, 0x4c)
+DECOMP_SIZE_ASSERT(Act1State::NamedTexture, 0x14)
+
+#define isle_asize(ARR) (sizeof(ARR) / sizeof((ARR)[0]))
+
+// Forward declarations
+Act1State::NamedTexture* ReadNamedTexture(LegoFile* p_file);
+void WriteNamedTexture(LegoFile* p_file, Act1State::NamedTexture* p_texture);
+void FUN_1003f540(LegoFile* p_file, const char* p_filename);
+
+enum {
```

Comment:
We generally declare enums inside the classes

Reply:
I didn't because it was not part of the public interface,  but I'll your suggestion.